annotation-web/src/main/java/eu/europeana/annotation/web/service/controller/admin/WhitelistRest.java

```diff
@@ -150,7 +150,7 @@ public ResponseEntity<String> loadDefaultWhitelist(
     }
 
     @DELETE
-    @RequestMapping(value = "/deleteall", method = RequestMethod.DELETE, produces = MediaType.APPLICATION_JSON_VALUE)
+    @RequestMapping(value = "/deleteall", method = RequestMethod.DELETE)
```

Comment:
the produces are needed for content negotiation mechanism

Reply:
Ok.